internal/services/machinelearning/machine_learning_workspace_network_outbound_rule_service_tag_resource.go

```diff
@@ -0,0 +1,344 @@
+				"AppService",
+				"AzureActiveDirectory",
+				"AzureAdvancedThreatProtection",
+				"AzureArcInfrastructure",
+				"AzureAttestation",
+				"AzureBackup",
+				"AzureBotService",
+				"AzureContainerRegistry",
+				"AzureCosmosDB",
+				"AzureDataLake",
+				"AzureDevSpaces",
+				"AzureInformationProtection",
+				"AzureIoTHub",
+				"AzureKeyVault",
+				"AzureManagedGrafana",
+				"AzureMonitor",
+				"AzureOpenDatasets",
+				"AzurePlatformDNS",
+				"AzurePlatformIMDS",
+				"AzurePlatformLKM",
+				"AzureResourceManager",
+				"AzureSignalR",
+				"AzureSiteRecovery",
+				"AzureSpringCloud",
+				"AzureStack",
+				"AzureUpdateDelivery",
+				"DataFactoryManagement",
+				"EventHub",
+				"GuestAndHybridManagement",
+				"M365ManagementActivityApi",
+				"M365ManagementActivityApi",
+				"MicrosoftAzureFluidRelay",
+				"MicrosoftCloudAppSecurity",
+				"MicrosoftContainerRegistry",
+				"PowerPlatformInfra",
+				"ServiceBus",
+				"Sql",
+				"Storage",
+				"WindowsAdminCenter",
+				"AppServiceManagement",
+				"AutonomousDevelopmentPlatform",
+				"AzureActiveDirectoryDomainServices",
+				"AzureCloud",
+				"AzureConnectors",
+				"AzureContainerAppsService",
+				"AzureDatabricks",
+				"AzureDeviceUpdate",
+				"AzureEventGrid",
+				"AzureFrontDoor.Frontend",
+				"AzureFrontDoor.Backend",
+				"AzureFrontDoor.FirstParty",
+				"AzureHealthcareAPIs",
+				"AzureLoadBalancer",
+				"AzureMachineLearning",
+				"AzureSphere",
+				"AzureWebPubSub",
+				"BatchNodeManagement",
+				"ChaosStudio",
+				"CognitiveServicesFrontend",
+				"CognitiveServicesManagement",
+				"DataFactory",
+				"Dynamics365ForMarketingEmail",
+				"Dynamics365BusinessCentral",
+				"EOPExternalPublishedIPs",
+				"Internet",
+				"LogicApps",
+				"Marketplace",
+				"MicrosoftDefenderForEndpoint",
+				"PowerBI",
+				"PowerQueryOnline",
+				"ServiceFabric",
+				"SqlManagement",
+				"StorageSyncService",
+				"WindowsVirtualDesktop",
+				"VirtualNetwork",
+			}, false),
+		},
+
+		"protocol": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ValidateFunc: validation.StringInSlice([]string{"*", "TCP", "UDP", "ICMP"}, false),
+		},
+
+		"port_ranges": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ValidateFunc: validation.StringIsNotEmpty,
+		},
+	}
+	return arguments
+}
+
+func (r WorkspaceNetworkOutboundRuleServiceTag) Attributes() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{}
+}
+
+func (r WorkspaceNetworkOutboundRuleServiceTag) Create() sdk.ResourceFunc {
+	return sdk.ResourceFunc{
+		Timeout: 30 * time.Minute,
+		Func: func(ctx context.Context, metadata sdk.ResourceMetaData) error {
+			var model machineLearningWorkspaceServiceTagOutboundRuleModel
+			if err := metadata.Decode(&model); err != nil {
+				return fmt.Errorf("decoding: %+v", err)
+			}
+
+			client := metadata.Client.MachineLearning.ManagedNetwork
+			subscriptionId := metadata.Client.Account.SubscriptionId
```

Comment:
Ordering here is not consistent with the vast majority of the provider, please order these correctly going forward
```suggestion
			client := metadata.Client.MachineLearning.ManagedNetwork
			subscriptionId := metadata.Client.Account.SubscriptionId
			
			var model machineLearningWorkspaceServiceTagOutboundRuleModel
			if err := metadata.Decode(&model); err != nil {
				return fmt.Errorf("decoding: %+v", err)
			}
```